File: WarshippyGame/Assets/MessagePacket.cs
using System;
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

[Serializable]
public class MessagePacket
{
    public int points;
    public string type_message;
    public string message;

    MessagePacket()
    {
        
    }

    public override String ToString()
    {
        return "Points: " + points + " Type: " + type_message + " Message: " + message;
    }

    public string ToJson()
    {
        return JsonUtility.ToJson(this);
    }
    
    public MessagePacket(int numeric_message, string message, string typeMessage)
    {
        this.points = numeric_message;
        this.message = message;
        this.type_message = typeMessage;
    }
    public MessagePacket(string message, string typeMessage)
    {
        this.points = 0;
        this.message = message;
        this.type_message = typeMessage;
    }
}

File: Python Server/elements/MessagePacket.py
import json


class MessagePacket:
    def __init__(self, message, type_message, points):
        self.dic = {
            "message": str(message),
            "type_message": str(type_message),
            "points": str(points)
        }

    def to_json(self, text):
        a = json.loads(text)
        self.type_message = a['type_message']
        self.message = a['message']
        self.points = a['points']

    def to_string(self):
        json_response = json.dumps(self.dic, ensure_ascii=False)
        return json_response

